app/src/lib/editors/win32.ts

```diff
@@ -144,6 +185,14 @@ function isExpectedInstallation(
   }
 }
 
+function getKeyOrEmpty(
+  keys: ReadonlyArray<RegistryValue>,
+  key: string
+): string {
+  const entry = keys.find(k => k.name === key)
+  return entry && entry.type === RegistryValueType.REG_SZ ? entry.data : ''
```

Comment:
For reference, `registry-js` does some lifting to map the registry value type to a JS type. I only implemented strings (`REG_SZ`) and number (`REG_DWORD`) for now, but we can add more in the future:

```
export type RegistryStringEntry = {
  readonly name: string
  readonly type: RegistryValueType.REG_SZ | RegistryValueType.REG_EXPAND_SZ
  readonly data: string
}

export type RegistryNumberEntry = {
  readonly name: string
  readonly type: RegistryValueType.REG_DWORD
  readonly data: number
}

// TODO: define some other shapes of data
export type RegistryValue = RegistryStringEntry | RegistryNumberEntry
```